src/graphql/queries/total-result-bets.js

```diff
@@ -27,19 +27,40 @@ module.exports = async (
   if (isNull(event)) throw Error('Event not found');
   const { numOfResults } = event;
 
+  // Accumulate all round 0 bets
+  let totalBetRound = await DBHelper.findBet({ eventAddress, eventRound: 0 });
+  const totalBets = accumulateBets(numOfResults, totalBetRound);
+
+  // Accumulate better round 0 bets
+  let betterBets
+  if (betterAddress) {
+    betterBetRound = await DBHelper.findBet({ eventAddress, eventRound: 0, betterAddress });
+    betterBets = accumulateBets(numOfResults, betterBetRound);
+  }
+
+  // Get result set
+  const resultSet = await DBHelper.findResultSet({ eventAddress, eventRound: 0 });
+  const { resultIndex, amount: resultSetAmount, centralizedOracleAddress } = resultSet;
+
   // Accumulate all result bets
-  let bets = await DBHelper.findBet({ eventAddress });
-  const resultBets = accumulateBets(numOfResults, bets);
+  let totalVoteRound = await DBHelper.findBet({ eventAddress, eventRound: { $gt: 0 } } );
+  const totalVotes = accumulateBets(numOfResults, totalVoteRound);
+
+  // Add result set amount
+  totalVotes[resultIndex] += resultSetAmount
```

Comment:
this wont work. `totalVotes` is an array of strings. suggest to just add the resultSet obj to totalVoteRound before calling accumulateBets()